lib/mustermann/grape/version.rb

```diff
@@ -1,7 +1,5 @@
-require 'mustermann/grape'
+# frozen_string_literal: true
 
-module Mustermann
-  class Grape
-    VERSION = '1.0.1'
-  end
+module MustermannGrape
+  VERSION = '1.0.1'
```

Comment:
(As you probably already considered), instead of changing the namespace, I think we can skip `require 'mustermann/grape'`. 
Of course, in that case, we cannot `require` `mustermann/grape` and`mustermann/grape/version` at the same time, though. I don't want to define a new namespace like `MustermannGrape`, but it's not strong opinion.

Reply:
The issue is that `Mustermann::Grape` inherits from `AST::Pattern`, so that brings a bunch of includes and causes all kinds of new problems.